src/x/config/config.go

```diff
@@ -42,37 +41,50 @@ var errNoFilesToLoad = errors.New("attempt to load config with no files")
 type Options struct {
 	DisableUnmarshalStrict bool
 	DisableValidate        bool
+
+	// LookupFunc -- see go.uber.org/config.Expand
```

Comment:
@mway thoughts on overall API here? Since this is a pretty thin wrapper, I could go the route of switching consumers to use the `config` package directly, and/or make this package look like more like the `config` package but also do validation, e.g.:

```
func Load(opts ...config.YAMLOption) {
   // load + validate
}
```

Reply:
i'd rather do it in stages - let's start by using `go.uber.org/config` under the hood, and then once that settles, remove `x/config`.

Reply:
Works for me!